Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM openjdk:11-jre-slim
+
+COPY docker/entrypoint.sh /usr/local/bin/
+RUN chmod a+x /usr/local/bin/entrypoint.sh
+
+WORKDIR /usr/local/fcrepo-camel-toolbox
+
+ARG FCREPO_CAMEL_TOOLBOX_VERSION=6.0.0-SNAPSHOT
+COPY fcrepo-camel-toolbox-app/target/fcrepo-camel-toolbox-app-${FCREPO_CAMEL_TOOLBOX_VERSION}-driver.jar driver.jar
+
+ENV FCREPO_CAMEL_TOOLBOX_HOME=/usr/local/fcrepo-camel-toolbox
```

Comment:
What could be use-cases where we need this environment variable?

Reply:
It's used by fcrepo-camel-toolbox to search for the property file (see https://github.com/docuteam/fcrepo-camel-toolbox/blob/fcrepo-3746/fcrepo-camel-common/src/main/java/org/fcrepo/camel/common/config/BasePropsConfig.java#L45).

@dbernstein Is FCREPO_CAMEL_TOOLBOX_HOME used anywhere else, e.g. to decide where to store LDPath Cache?